src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -10,22 +10,32 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 
-<Project>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Namespace not needed.

Reply:
Sure, but full framework MSBuild will complain if there is no namespace. Can you confirm that this props ships/imports only with Core MSBuild?